schema/insert_metadata.py

```diff
@@ -0,0 +1,177 @@
+import pandas as pd
+import requests
+from pymssql import connect, Connection
```

Comment:
Small detail, make sure to order libraries properly, Dan will not be happy with this 😤